test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -999,6 +1007,29 @@ def stop_proc_monitor(self):
         self.metrics_data['procs'] = self._procmon.db_proc_info
         self._process_monitoring = False
 
+    def sched_configure(self, sched_name, sched_home=None):
+        """
+        Start scheduler with creating required directories for scheduler
+        :param sched_name: scheduler name
+        :type sched_name: str
+        :param sched_home: path of scheduler home and log directory
+        :type sched_home: str
+        """
+        pbs_home = self.server.pbs_conf['PBS_HOME']
+        if sched_home is None:
+            sched_home = pbs_home
+        sched_priv_dir = 'sched_priv_' + sched_name
+        sched_logs_dir = 'sched_logs_' + sched_name
+        if not os.path.exists(os.path.join(sched_home, sched_priv_dir)):
+            self.du.run_copy(self.server.hostname,
+                             os.path.join(pbs_home, 'sched_priv'),
+                             os.path.join(sched_home, sched_priv_dir),
+                             recursive=True)
+        if not os.path.exists(os.path.join(sched_home, sched_logs_dir)):
+            self.du.mkdir(path=os.path.join(sched_home, sched_logs_dir),
+                          sudo=True)
+        self.server.schedulers[sched_name].start(sched_home)
```

Comment:
I have self.scheds , But the dict key is the hostname or the server name and then the values is again a dictionary of scheduler names and its objects.
So it has to something like self.schedulers[hostname][sched_name] . If this is okay i will change it.

Reply:
The dict of self.schedulers is consistent with self.moms.  It's a dictionary whose keys are hostnames.  It would make sense that we make it a dictionary of scheduler names as well.  Since most tests will only use schedulers associated with one server, it might make sense to also create a shorthand for one server.  This way you can do something like self.scheds['name'] and it means self.schedulers[self.mom.shortname]['name']

Reply:
I was not using self.scheds for anything other than intializing and iterating through hostnames .
Now i have created a shorthand for self.server.schedulers and will update all the test cases.